assets/js/atomic/blocks/product-elements/price/edit.tsx

```diff
@@ -7,17 +7,38 @@ import {
 	BlockControls,
 	useBlockProps,
 } from '@wordpress/block-editor';
-import { __ } from '@wordpress/i18n';
 import { useEffect } from 'react';
 
 /**
  * Internal dependencies
  */
 import Block from './block';
 import withProductSelector from '../shared/with-product-selector';
-import { BLOCK_TITLE, BLOCK_ICON } from './constants';
+import {
+	BLOCK_TITLE as label,
+	BLOCK_ICON as icon,
+	BLOCK_DESCRIPTION as description,
+} from './constants';
+
+interface Attributes {
+	textAlign: 'left' | 'center' | 'right';
+}
+
+interface Context {
+	queryId: number;
+}
+
+interface Props {
+	attributes: Attributes;
+	setAttributes: ( attributes: Record< string, unknown > ) => void;
```

Comment:
Maybe we can use `BlockAttributes` from `./types` file as shown below. This will help with autocompletion. What do you think?

```suggestion
	setAttributes: ( attributes: Partial< BlockAttributes > & Record< string, unknown > ) => void;
```

Reply:
Good catch, @imanish003. Let's definitely add `Partial< BlockAttributes >` here.